### PR TITLE
feat(auth-admin-api): Client create scope endpoint

### PR DESCRIPTION
**Pull request by snaerseljan**
# Client create scope endpoint

## What

This PR includes creating scope for specific tenant in auth-admin-api

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

**Review by GunnlaugurG — APPROVED** [review]
LGTM !

File: apps/services/auth/admin-api/src/app/v2/scopes/test/me-scopes.spec.ts [hunk 1]
```diff
@@ -1,7 +1,15 @@
 import request from 'supertest'
 
 import { AdminPortalScope } from '@island.is/auth/scopes'
-import { AdminScopeDTO, SequelizeConfigService } from '@island.is/auth-api-lib'
+import {
+  AdminScopeDTO,
+  ApiScopeUserClaim,
+  Client,
+  clientBaseAttributes,
+  ClientGrantType,
```
[the remark this answers is not in the document]
Comment (snaerseljan) [reply apps/services/auth/admin-api/src/app/v2/scopes/test/me-scopes.spec.ts]: Nice catch. Man we really need an eslint rule for unused imports

File: libs/shared/utils/src/lib/validateClientId.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
+/**
+ * Validates that the client id is prefixed with the tenant and that it matches the regex
+ * Value can only contain alphanumeric characters, hyphens, underscores, periods and forward slashes.
+ * @returns true if valid, false otherwise
+ */
+export const validateClientId = ({
```
Comment (eirikurn) [inline libs/shared/utils/src/lib/validateClientId.ts:6]: This is quite auth related. Wonder how it fits into the naming/folder convention.

`libs/auth/shared`?

Comment (snaerseljan) [reply libs/shared/utils/src/lib/validateClientId.ts:6]: I found it hard to find a correct place for this. Creating new lib is a good idea!

**Pull request merged**